Utilities/bootstrap

```diff
@@ -582,15 +598,22 @@ def build_swiftpm_with_cmake(args):
 
 def build_swiftpm_with_swiftpm(args, integrated_swift_driver):
     """Builds SwiftPM using the version of SwiftPM built with CMake."""
-    note("Building SwiftPM (with swift-build)")
 
     swiftpm_args = [
         "SWIFT_EXEC=" + args.swiftc_path,
         "SWIFT_DRIVER_SWIFT_EXEC=" + args.swiftc_path,
-        "SWIFTPM_PD_LIBS=" + os.path.join(args.bootstrap_dir, "pm"),
-        os.path.join(args.bootstrap_dir, "bin/swift-build"),
-        "--disable-sandbox",
     ]
+
+    if args.bootstrap:
+        note("Building SwiftPM (with a freshly built swift-build)")
+        swiftpm_args.append("SWIFTPM_PD_LIBS=" + os.path.join(args.bootstrap_dir, "pm"))
+        swiftpm_args.append(os.path.join(args.bootstrap_dir, "bin/swift-build"))
+    else:
+        note("Building SwiftPM (with a prebuilt swift-build)")
+        swiftpm_args.append(os.path.join(os.path.split(args.swiftc_path)[0], "swift-build"))
+
+    swiftpm_args.append("--disable-sandbox")
```

Comment:
Why does the sandbox need to be disabled?  This is Darwin specific and changes the behavior no?

Reply:
No idea why that flag is applied, I simply copied it here from above. Since I use a different path to swift-build, I had to break it out here so it comes after swift-build. This flag has always been applied when building SPM, I haven't changed anything regarding that flag in this pull.

Reply:
Oh, bleh, I didn't see that.  Thanks!

Reply:
Not sure about this either, but in the context of this PR it seems right to leave as is.